### PR TITLE
DOCS-942: Add MoveOnGlobe() to the Motion Service docs

### PR DESCRIPTION
**Pull request by sguequierre**
Implements the second segment of the changes outlined here: [Docs Changes for GPS Navigation](https://docs.google.com/document/d/1ReL7RU0FoaAe8gyPbMgMovHpgdx3Jy4c3R0LFCVWkKI/edit) (section below `"Changes to https://docs.viam.com/services/motion/"`)

* Add the MoveOnGlobe() method to the Motion service

### CODE REVIEW

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:294]: I moved this down so that the methods would align with the order shown in the table, and it makes sense comprehensively to group the endpoints that produce motion together

File: docs/program/apis/_index.md
Comment (sguequierre) [inline docs/program/apis/_index.md:371]:
```suggestion
[`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
```

File: docs/program/apis/_index.md
Comment (sguequierre) [inline docs/program/apis/_index.md:371]: @npentrel I was originally thinking `Move a component to a specific latitude and longitude, using a [Movement Sensor](/components/movement-sensor/) to determine the location.` for this line but i think that could be a bit overwhelming. Thought this was easier to parse, suggestions welcome

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:574]:
```suggestion
// Define a destination Point at the GPS coordinates [0, 0]
```

File: docs/program/apis/_index.md
Comment (tessavitabile) [inline docs/program/apis/_index.md:371]: Should add that MoveOnGlobe (and MoveOnMap) is only supported for bases.

File: docs/services/motion/_index.md
Comment (tessavitabile) [inline docs/services/motion/_index.md:516]: We do not plan to support translation, since you can already specify the location using the latitude and longitude.

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:499]:
```suggestion
`MoveOnGlobe()` is intended for use with the [Navigation Service](/services/navigation/), providing autonomous GPS navigation for rover [bases](/components/base/).
```

**Review by tessavitabile — APPROVED** [review]
LGTM mod comments

I assume we're deferring saying that MoveOnGlobe will respect the minimum turning radius from GetProperties until GetProperties has been documented.

File: docs/services/motion/_index.md
Comment (tessavitabile) [inline docs/services/motion/_index.md:513]: MoveOnGlobe is not experimental. Only specifying "heading" when the minimum turning radius is greater than zero is experimental.

File: docs/program/apis/_index.md
Comment (npentrel) [reply docs/program/apis/_index.md:371]: sounds good to me.

File: docs/services/motion/_index.md
Comment (npentrel) [inline docs/services/motion/_index.md:48]:
```suggestion
[`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
```

**Review by npentrel — APPROVED** [review]
LGTM once you've addressed comments & Tess approved

File: docs/program/apis/_index.md
Comment (sguequierre) [inline docs/program/apis/_index.md:371]:
```suggestion
[`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a [base](/component/base/) component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
```

File: docs/program/apis/_index.md
Comment (sguequierre) [inline docs/program/apis/_index.md:370]:
```suggestion
[`MoveOnMap`](/services/motion/#moveonmap) | Move a [base](/component/base/) component to a `Pose` in respect to the origin of a [SLAM](/services/slam/) map.
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:47]:
```suggestion
[`MoveOnMap`](#moveonmap) | Move a [base](/components/base/) component to a `Pose` in respect to the origin of a [SLAM](/services/slam/) map.
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:48]:
```suggestion
[`MoveOnGlobe`] | Move a [base](/components/base/) component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:432]:
```suggestion
Move a [base](/component/base/) component to a [`Pose`](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Pose) in respect to the origin of a [SLAM](/services/slam/) map.
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:439]:
```suggestion
- `component_name` ([ResourceName](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.ResourceName)): The `"name"` of the base you want to move.
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:467]:
```suggestion
- `componentName` [(resource.Name)](https://pkg.go.dev/go.viam.com/rdk/resource#Name): The `"name"` of the base to move.
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:494]:
```suggestion
Move a [base](/components/base/) component to a destination GPS point, represented in geographic notation *(latitude, longitude)*.
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:516]:
```suggestion
Translation in obstacles is not supported by the [Navigation Service](/services/navigation/).
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:518]:
```suggestion
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:559]:
```suggestion
- `componentName` [(resource.Name)](https://pkg.go.dev/go.viam.com/rdk/resource#Name): The `"name"` of the base to move.
```

File: docs/program/apis/_index.md
Comment (sguequierre) [reply docs/program/apis/_index.md]: Done

File: docs/services/motion/_index.md
Comment (sguequierre) [reply docs/services/motion/_index.md]: Ok awesome. Making this clear

File: docs/program/apis/_index.md
Comment (sguequierre) [inline docs/program/apis/_index.md:370]:
```suggestion
[`MoveOnMap`](/services/motion/#moveonmap) | Move a [base](/components/base/) component to a `Pose` in respect to the origin of a [SLAM](/services/slam/) map.
```

File: docs/program/apis/_index.md
Comment (sguequierre) [inline docs/program/apis/_index.md:371]:
```suggestion
[`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a [base](/components/base/) component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
```

File: docs/services/motion/_index.md
Comment (sguequierre) [inline docs/services/motion/_index.md:432]:
```suggestion
Move a [base](/components/base/) component to a [`Pose`](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Pose) in respect to the origin of a [SLAM](/services/slam/) map.
```

**Pull request merged**
